sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -36,7 +36,7 @@ static uint16_t min_seq_expected = 0;
 static uint16_t max_seq_expected = 0;
 static char ipv6_str[IPV6_ADDR_MAX_STR_LEN];
 
-static void usage(char **argv)
+static void usage(char * *argv)
```

Comment:
This is an error in the uncrustify config. Already worked a fix out. For your PR it suffices to correct it manually.